include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 18
+#define SOF_ABI_MINOR 21
 #define SOF_ABI_PATCH 0
```

Comment:
Did we forgot to bump the ABI on Linux side in the past?

Reply:
yes, looks like  we need a dedicated person to align with me for FW side.